src/main/resources/assets/mekanism/lang/de_de.lang

```diff
@@ -385,9 +385,11 @@ upgrade.muffling=Dämpfung
 upgrade.muffling.desc=Verringert den von Maschinen produzierten Schall.
 
 # Key description text
-mekanism.key.mode=Moduswechsel
-mekanism.key.armorMode=Küstungsmoduswechsel
+mekanism.key.mode=Item Modus wechsel
+mekanism.key.armorMode=Rüstungs Modus wechsel
+mekanism.key.armorMode=Fuß Modus wechsel
```

Comment:
You have the same lang key `mekanism.key.armorMode` here twice

Reply:
oops i guess i know how that got there...